it/repositories/FileNotificationRepositoryISpec.scala

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package repositories
+
+import models.SDESNotificationRecord
+import models.notification._
+import org.mongodb.scala.bson.collection.immutable.Document
+import org.mongodb.scala.result.DeleteResult
+import org.scalatest.matchers.should.Matchers._
+import play.api.test.Helpers._
+import utils.IntegrationSpecCommonBase
+
+import scala.concurrent.Future
+
+class FileNotificationRepositoryISpec extends IntegrationSpecCommonBase {
+  lazy val repository: FileNotificationRepository = injector.instanceOf[FileNotificationRepository]
+
+  def deleteAll(): Future[DeleteResult] =
+    repository
+      .collection
+      .deleteMany(filter = Document())
+      .toFuture
+
+  class Setup {
+    await(deleteAll())
+  }
+
+  val notification1:SDESNotification = SDESNotification(informationType = "info",
+    file = SDESNotificationFile(
+      recipientOrSender = "penalties",
+      name = "ame", location = "someUrl", checksum = SDESChecksum(algorithm = "sha", value = "256"), size = 256, properties = Seq.empty[SDESProperties]
+    ), audit = SDESAudit("file 1"))
+
+  val notification2:SDESNotification = notification1.copy(audit = SDESAudit("File 2"))
+
+  "insertFileNotifications" should {
+    "insert a single notification received from the backend" in new Setup{
+      val result = await(repository.insertFileNotifications(Seq(notification1)))
+      result shouldBe true
+
+      val recordsInMongoAfterInsertion:Seq[SDESNotificationRecord] = await(repository.collection.find()toFuture)
+      recordsInMongoAfterInsertion.size shouldBe 1
+      recordsInMongoAfterInsertion.head.notification shouldBe notification1
+    }
+
+    "insert multiple notifications received from the backend" in new Setup {
+      val result = await(repository.insertFileNotifications(Seq(notification1, notification2)))
+      result shouldBe true
+
+      val recordsInMongoAfterInsertion:Seq[SDESNotificationRecord] = await(repository.collection.find()toFuture)
+      recordsInMongoAfterInsertion.size shouldBe 2
+      recordsInMongoAfterInsertion.head.notification shouldBe notification1
+      recordsInMongoAfterInsertion(1).notification shouldBe notification2
```

Comment:
This is picky, but you could do .last instead (up to you):
```suggestion
      recordsInMongoAfterInsertion.last.notification shouldBe notification2
```